### DISCUSSION TITLE
Thank you

### DISCUSSION THREAD

**Discussion by ValentinVignal** [rydmike/flex_color_picker#56] [Q&A] [answered]
I wanted to thank you for this awesome library and the amount of effort you put in the documentation. The live demo is also incredible and helped us a lot.

You can close this when you read it.

Thank you!

**Comment by rydmike** [comment, marked as answer]
Hi @ValentinVignal, always glad to hear back from and get a **thank you** from an **actual FlexColorPicker** package users! :)

Glad you like the docs too. I do plan to move them and the docs for my other more well known public package [**FlexColorScheme**](https://pub.dev/packages/flex_color_scheme) to their own separate doc sites, using [docs.pag](https://docs.page/homepage) it should be simple enough, like e.g. [GoRouter](https://pub.dev/packages/go_router) does.

Doing so should make the docs for both packages even easier to read/follow, and much simpler to maintain and add to, without the need to also release a new version of the package for simple documentation updates and improvements.

Using live demos to show of a package capability is a really cool thing that you can do with Flutter Web. I think package authors should do that much more, even always if it is a more complex UI widget at least. While UI widget APIs are generally simple, discovering all the options can be a bit tricky sometimes.

With **FlexColorScheme**, I went a step further, with the 5 step tutorial where the 5th step is a ["ThemesPlayground"](https://rydmike.com/flexcolorschemeV4Tut5) app, where you can configure and try almost all API features and you can generate the config code for any setup you are looking at, and just paste it into your app. A really great way to discover the API and get started and just fine tune further in code. I might enhance the ColorPicker web demo with similar features too later as well.

Working on version 5 of **FlexColorScheme** now, lots of [changes in progress](https://github.com/rydmike/flex_color_scheme/blob/version-5-0-0-dev1/CHANGELOG.md), but minimal breaking so far. For it I am going to need, well would like to have, Material 3 tonal palettes, in the ColorPicker that is used in the ThemesPlayground app,  so I will probably add this [feature soon](https://github.com/rydmike/flex_color_picker/issues/35) as a none breaking one  to past API, at least I think that should be possible and quite easy.

There are bunch of other [enhancements listed on my TODOs](https://github.com/rydmike/flex_color_picker/issues?q=is%3Aopen+is%3Aissue+label%3Aenhancement) for the ColorPicker. If you have any favorite ones you would like me to prioritize, go give it a 👍🏻 and maybe a comment on any specifics regarding it.
